src/utils/format-message-for-whatsapp.ts

```diff
@@ -0,0 +1,10 @@
+export function formatMessageForWhatsApp(
+  numberInInternationalFormat: string,
+  message: string
+) {
+  const baseURL = "https://wa.me"
```

Comment:
Poderiamos mover pra fora da função e usarmos em maiuscula para sinalizar que é constante.
```suggestion
  const WHATSAPP_BASE_URL = "https://wa.me"
```